db/schema.rb

```diff
@@ -193,6 +193,7 @@
     t.boolean  "approved",               default: false
     t.string   "github_access_token"
     t.boolean  "admin",                  default: false
+    t.integer  "submission_form_type",   default: 0,     null: false
```

Comment:
I'd rather use something like [Storext](https://github.com/G5/storext) ([tutorial](https://medium.com/@mscccc/rails-storing-user-settings-2016-edition-443c5ea5a921#.4qn0e833n)) than adding a separate column for each possible user preference in the future. Seems to be helpful with email notifications and course settings as well.